src/test/java/gmailgdogra/ExtractOfficersTest.java

```diff
@@ -24,7 +29,18 @@
 
     @Test
     void from() throws IOException {
-        List<SwipeRecord> swipeRecords = ReadXlsx.parse("src/main/resources/testFile.xlsx");
+        Path path = Paths.get("src/main/resources/testFile.xlsx");
+        String name = "testFile.xlsx";
+        String originalFileName = "testFile.xlsx";
+        String contentType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
+        byte[] content = null;
+        try {
+            content = Files.readAllBytes(path);
+        } catch (final IOException ignored) {
```

Comment:
Ignored exceptions bother people. You coul've failed the test with a meaningful message instead.

Reply:
changed to 
```
try {
            content = Files.readAllBytes(path);
        } catch (IOException e) {
            Assertions.fail("Unable to read file at " + path);
        }
```